src/app/helpers/controller/cms-mixin.js

```diff
@@ -50,7 +46,7 @@ async function getUrlFor(slug) {
 
     // A little magic to handle the news slug
     if (slug === 'news') {
-        apiUrl = await newsPromise;
+        apiUrl = '/apps/cms/api/pages/openstax-news';
```

Comment:
is it possible to use the API base from the settings file (I think you have this somewhere)? That way there is only one place to change the API path if it changes.. again.

Reply:
Good catch